web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,69 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
```

Comment:
It is not clear (at least for me) from the rest of the doc, if each user should change password for himself or if one user should change password for all the mentioned users, ...or something between...

Reply:
This is only one form where user can change his password. I've made appropriate changes.